Libplanet/Blocks/BlockHeader.cs

```diff
@@ -196,6 +210,11 @@ public Bencodex.Types.Dictionary ToBencodex()
                 .Add(NonceKey, Nonce.ToArray())
                 .Add(HashKey, Hash.ToArray());
 
+            if (ProtocolVersion != 0)
+            {
+                dict = dict.Add(ProtocolVersionKey, ProtocolVersion);
```

Comment:
How about remove if statement and force to record protocol version afterward?

Reply:
It has to accept the field being omitted for backward compatibility (which is the point of the [spec][1]).

[1]: https://github.com/planetarium/libplanet/issues/1142

Reply:
👌